Running `make deploy` returns an error

**Describe the bug**
Followed the installation steps and running `make deploy` returns an error.

```text

➜  dast-operator git:(master) ✗ make deploy
cd config/manager && kustomize edit set image controller=reg.captainjustin.space/banzaicloud/dast-operator:latest
kustomize build config/default | kubectl apply -f -
namespace/dast-operator-system unchanged
error: error validating "STDIN": error validating data: [ValidationError(CustomResourceDefinition.spec.validation.openAPIV3Schema.properties.spec.properties.analyzer.properties.service.properties.spec.properties.ports): unknown field "x-kubernetes-list-map-keys" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.JSONSchemaProps, ValidationError(CustomResourceDefinition.spec.validation.openAPIV3Schema.properties.spec.properties.analyzer.properties.service.properties.spec.properties.ports): unknown field "x-kubernetes-list-type" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.JSONSchemaProps]; if you choose to ignore these errors, turn validation off with --validate=false
Makefile:43: recipe for target 'deploy' failed
make: *** [deploy] Error 1

```



**Steps to reproduce the issue:**

Running on Kubernetes 1.15.12

```bash

kubectl create namespace cert-manager
helm repo add jetstack https://charts.jetstack.io
helm repo update
kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.1/cert-manager.crds.yaml
helm install cert-manager jetstack/cert-manager --namespace cert-manager --version v0.15.1
git clone https://github.com/banzaicloud/dast-operator.git
cd dast-operator
make deploy
```



**Expected behavior**
Should not error out and crash the install

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->

**Additional context**
<!-- Add any other context about the problem like release version, branch, etc. -->

--- a/Makefile
+++ b/Makefile
@@ -41,7 +41,7 @@ uninstall: manifests
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy:
 	cd config/manager && kustomize edit set image controller=${IMG}
-	kustomize build config/default | kubectl apply -f -
+	kustomize build config/default | kubectl apply --validate=false -f -
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen

